### PR TITLE
DONT MERGE: Eng 10207 pause mode

### PR DESCRIPTION
**Pull request by akhanzode**

### CODE REVIEW

File: src/frontend/org/voltdb/Inits.java [hunk 1]
```diff
@@ -570,6 +570,10 @@ public void run() {
                 adminPort = m_config.m_adminPort;
             // other places might use config to figure out the port
             m_config.m_adminPort = adminPort;
+            //Allow cli to set admin mode otherwise use whats in deployment for backward compatibility
+            if (m_config.m_isPaused.get()) {
+                m_rvdb.setStartMode(OperationMode.PAUSED);
```
Comment (vtkstef) [inline src/frontend/org/voltdb/Inits.java]: shouldn't we do this in VoltDB.Configuration instead? if not why here and not there

File: src/frontend/org/voltdb/sysprocs/Pause.java [hunk 2]
```diff
@@ -61,6 +61,7 @@ public DependencyPair executePlanFragment(
                 VoltDB.instance().getHostMessenger().getZK().setData(
                         VoltZK.operationMode,
                         OperationMode.PAUSED.getBytes(), -1);
+                VoltDB.instance().getHostMessenger();
```
Comment (vtkstef) [inline src/frontend/org/voltdb/sysprocs/Pause.java]: why are you 'just' getting the host messenger?

File: src/frontend/org/voltdb/Inits.java (hunk 1 above)
Comment (akhanzode) [inline src/frontend/org/voltdb/Inits.java]: I can remove it actually dont need it.

File: src/frontend/org/voltdb/sysprocs/Pause.java (hunk 2 above)
Comment (akhanzode) [inline src/frontend/org/voltdb/sysprocs/Pause.java]: Will remove..

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 3]
```diff
@@ -2714,7 +2718,10 @@ public void onRestoreCompletion(long txnId, Map<Integer, Long> perPartitionTxnId
             // Shouldn't be here, but to be safe
             m_mode = OperationMode.RUNNING;
         }
-        consoleLog.l7dlog( Level.INFO, LogKeys.host_VoltDB_ServerCompletedInitialization.name(), null);
+        String mstr = (m_mode == OperationMode.PAUSED) ? "PAUSED" : "NORMAL";
+        Object args[] = { mstr };
```
Comment (vtkstef) [inline src/frontend/org/voltdb/RealVoltDB.java]: maybe

```java
Object args[] = { m_mode == OperationMode.PAUSED) ? "PAUSED" : "NORMAL" };
```

Comment (akhanzode) [inline src/frontend/org/voltdb/RealVoltDB.java]: Sure..

**Comment by akhanzode** [comment]
squash merged.

**Pull request closed without merging**